### ISSUE TITLE
The Slider component does not have the relevant attributes required to properly inform screen reader users of the current set value

### ISSUE THREAD

**Issue by nicklemmon** [SparkPost/matchbox#201]
The `<Slider/>` component is not currently constructed 100% properly from an a11y POV.

See the following article from MDN:
[Using the aria-valuemin attribute](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-valuemin_attribute)

## Issues

- [X] The slider current lacks `role="slider"` - _Note:_ Looks like I just missed this on first pass as it was already in place.
- [ ] The slider currently lacks the `aria-valuenow` attribute that informs a screen reader user of the current value of the slider

The slider *is* currently controllable with a keyboard so that's good news. Fixing the missing HTML attributes is relatively straightforward in comparison.

**Comment by jonambas** [comment]
Should the attributes be moved up to the wrapper?
I do see a bug though, `aria-valuenow` isn't updating correctly with `value`

This is the current structure:
```js
    <div>      // Wrapper
      <div />  // Rail
      <div />  // Highlighted Track
      <div     // Handle
        id={id}
        aria-valuemin={min}
        aria-valuemax={max}
        aria-valuenow={value}
        aria-disabled={disabled}
        className={styles.Handle}
        onBlur={onBlur}
        onFocus={onFocus}
        onKeyDown={disabled ? noop : handleKeyDown}
        role='slider'
        style={{ left: sliderLocation }}
        tabIndex='0'
      />
    </div>
```

**Comment by nicklemmon** [comment]
It looks like the aria attributes are in the right place - they belong on the element that can receive focus so that a screen reader user is informed of component's state when they focus that particular element.

So looks like just fixing `aria-valuenow` is the only thing necessary!

**Comment by jonambas** [comment]
👍 Will add this to #183 / UX-137
